src/backend/common/DefaultMemoryManager.cpp

```diff
@@ -325,7 +332,7 @@ void DefaultMemoryManager::userLock(const void *ptr) {
 
     lock_guard_t lock(this->memory_mutex);
 
-    locked_iter iter = current.locked_map.find(const_cast<void *>(ptr));
+    auto iter = current.locked_map.find(const_cast<void *>(ptr));
```

Comment:
nit: I am aware this variable is not used in a very large scope, but it could also use a better name to help while reading the code.